source/domain-migration.txt

```diff
@@ -0,0 +1,46 @@
+================
+Domain Migration
+================
+
+The App Services base URL will be updated from ``https://realm.mongodb.com/``
+to a new URL: ``https://services.cloud.mongodb.com``.
+
+This change applies to the App Services UI, Admin API requests, and client
+API requests through the Atlas Device SDKs.
+
+Relevant SDK versions
+---------------------
+
+Starting in the following versions, the SDKs will use the new base URL by
+default:
+
+- C++ SDK: version TBD
+- Flutter SDK: version TBD
+- Java SDK: version TBD
+- Kotlin SDK: version TBD
+- .NET SDK: version TBD
+- Node.js SDK: version TBD
+- React Native SDK: version TBD
+- Swift SDK: version TBD
+
+Begin Migration Now
```

Comment:
If we break this section out into the relevant places where updates may need to occur, that would make it easier to scan and make sure people don't miss updating in important spots. i.e. something like:

> Begin Migration Now
> ---------------------
> 
> The new URL...
> 
> Client Applications
> ~~~~~~~~~~~~~~
> 
> Begin using the new URL by specifying it as a base URL in your App Configuration. Refer to the SDK docs for details...
> 
> Domain Access List
> ~~~~~~~~~~~~~~~
> 
> If you use a domain access list, add the new domain to this list to avoid service disruptions:
> 
> ```
> *.services.cloud.mongodb.com
> ```
> 
> Region-Specific Base URLs
> ~~~~~~~~~~~~~~~~~~~~~
> 
> If your app uses a local region instead of the global endpoint:
> 
> ```
> https://us-east-1.aws.realm.mongodb.com/
> ```
> 
> Update the local region domain in your App Configuration's baseURL to avoid service disruptions:
> 
> ```
> https://us-east-1.aws.services.cloud.mongodb.com
> ```

Reply:
Excellent feedback. Thanks, Dachary!